{{cookiecutter.project_slug}}/Makefile

```diff
@@ -60,14 +62,16 @@ setup: ### Setup remote environment
 		--name $(SETUP_NAME) \
 		--preset cpu-small \
 		--detach \
-		--volume $(PROJECT_PATH_STORAGE):$(PROJECT_PATH_ENV):ro \
+		--volume $(PROJECT_PATH_STORAGE):$(PROJECT_PATH_ENV):rw \
```

Comment:
`pip install -e .` writes files to the project directory.

Reply:
not the case anymore. Still, I think, there's no point of mounting project root as `ro` during installation -- Let's not mount it then.